guava/src/com/google/common/collect/Queues.java

```diff
@@ -419,4 +421,181 @@ private Queues() {}
   public static <E> Deque<E> synchronizedDeque(Deque<E> deque) {
     return Synchronized.deque(deque, null);
   }
+  
+  /**
+   * Creates a reversed view of {@code deque}.
+   * 
+   * @param deque the deque to be wrapped in a reversed view
+   * @return a reversed view of the specified deque
+   */
+  public static <E> Deque<E> reversedDeque(Deque<E> deque) {
+    if (deque instanceof ReversedDeque) {
+      return ((ReversedDeque<E>)deque).deque;
```

Comment:
If you access this field, you should declare the deque field package private to prevent generation of a synthetic accessor.

Reply:
Or use `delegate()`, which is already accessible as it's `protected`.

Reply:
Or, even better I imagine, call `delegate()` instead? :)

Reply:
@Maaartinus got there before me. 😜